client/camera/detectPrograms.js

```diff
@@ -398,6 +403,19 @@ export default function detectPrograms({ config, videoCapture, dataToRemember, d
     }
   });
 
+  // all points which haven't been matched to a shape are added as markers to the paper data
+  const markers = keyPoints
+    .filter(({ matchedShape }) => !matchedShape)
+    .map(({ colorIndex, avgColor, pt, size }) => {
+      const { x, y } = projectPointToUnitSquare(pt, videoMat, config.knobPoints);
+      return { size, position: { x, y }, avgColor, colorIndex };
```

Comment:
`colorIndex` might not be super meaningful to programs. How about omitting that and changing `avgColor` to simply `color`?

Reply:
Ah I see how you used the `colorIndex` in the curling game and how that was convenient. Hm, I could get on board with that. Though might be nicer then to convert it to a colour name ("red", "green", etc)?

Reply:
Agree with you, I'll change that.